src/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs

```diff
@@ -14,27 +16,102 @@ namespace System.Net.Security.Tests
 {
     public class SslStreamStreamToStreamTest
     {
+        private readonly byte[] sampleMsg = Encoding.UTF8.GetBytes("Sample Test Message");
+        private const int TaskWaitTime = 15; // seconds
+
+        #region Tests
```

Comment:
Regions not necessarily are code smell. Infact they are used to organize the code in a better way for the sake of readability. While putting more classes in single class & making it a master sort of class is definitely a bad design. I will take out the stream & network classes out. to another files